docs/advanced/command-line/files/sources.md

```diff
@@ -0,0 +1,36 @@
+# Sources
+
+The file describing how to access the sources, passed as an argument to the -s command line option
+
+## Format of the file
+We have provided an example of how the contents of the file should be structured in **sources.csv** in the Manual Books example.
```

Comment:
Instead of 'we' maybe we can keep it more general like "An example has been provided of...etc."

Reply:
Done!